ibm_catalog.json

```
@@ -0,0 +1,122 @@
+{
+    "products": [
+        {
+            "features": [
+                {
+                    "title": "Supports multi-factor authentication",
+                    "description": "Supports multi-factor authentication (MFA) for all users or federated users. Includes email-based, TOTP, and U2F MFA."
+                },
+                {
+                    "title": "Supports general IAM settings",
+                    "description": "User can configure the following IAM Account settings:\n1. Restrict API Key creation. \n2. Restrict service ID creation\n 3. Customize session activity and inactivity timeouts (in seconds). \n4. Customize Access and Refresh token expiration (in seconds). \n5. Global Shell Settings. \n6. Public Access Group."
+                },
+                {
+                    "title": "Restrict IP address access",
+                    "description": "User can restrict specified IP subnets or IP addresses. \nIt also supports the following two control modes: \n1. Monitor\n2. Restrict"
```

```suggestion
                    "description": "Restrict IP subnets and IP addresses. \nSupports both "monitor" and "restrict" control modes"
```

I can't find any hits in the Cloud docs for "control mode" or `monitor mode` or `restrict mode`. We should use the standard IAM terms for these things, but I don't know what those are.

---

Its here https://github.com/terraform-ibm-modules/terraform-ibm-iam-account-settings/blob/main/main.tf#L72